sandbox/grommet-app/src/pages/layouts/kinds/RecordDetail/JobList.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  List,
+  NameValueList,
+  NameValuePair,
+  Text
+} from 'grommet';
+import { Edit, Trash } from 'grommet-icons';
+import { sentenceCase } from "../../../../utils/format";
+
+export const JobList = (
+  { jobs }:
+    {
+      jobs: {
+        id: string,
+        name: string,
+        [key: string]: any,
+      }[]
+    }
+) => {
+  return (
+    <List
+      data={jobs}
+      defaultItemProps={{ pad: { horizontal: 'none', vertical: 'xsmall' } }}
+    >
+      {(datum) => (
+        <Box
+          key={datum.id}
+          background="background-contrast"
+          round="small"
+          pad={{ left: 'small', right: 'xsmall', vertical: 'small' }}
+        >
+          <Box direction="row" justify="between">
+            <Text>{datum.name}</Text>
+            <Box direction="row">
+              <Button icon={<Edit />} size="small" />
```

Comment:
<img width="608" alt="Screenshot 2025-01-27 at 9 38 19 PM" src="https://github.com/user-attachments/assets/eadae40b-3552-4dbb-8612-b0ac292b693d" />

The spacing/text hierarchy feels awkward here. Feels like we could use a bit more spacing below "Update firmware"

Reply:
Agree.  Fixed.